content/topics/node-and-sql/_index.md

```diff
@@ -2,4 +2,24 @@
 _db_id: 44
 content_type: topic
 title: Introduction to Node and SQL
----
\ No newline at end of file
+---
+
+#### In this topic, you will learn how to connect to a PostgreSQL database using node
+
+- node-Postgres is a collection of Node.js modules used to interact with a PostgreSQL database.
+- node-Postgres has many features like callbacks, connection pooling, prepared statements, async/await, and promises.
+- connection pooling allows your application to reuse existing database connections by automatically saving the connection to a pool so it can be reused instead of having to create a new connection every time data is being retrieved/accessed. An example of this can be a cache component where data is frequently accessed and stored.
+
+### Connection pooling
+
+#### Client
+
+- [Node & SQL tutorial using Client](https://node-postgres.com/api/client)
+
+#### Pool
+
+- [Node & SQL tutorial using Pool](https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-node-js-on-ubuntu-20-04)
+
+### JavaScript learn-callbacks-promises-async-await
```

Comment:
This heading is not very good.

Reply:
It's important to get things in the right order as well. This is a prerequisite. So it should be at the top of the file.

Eg:

```
## Prerequisites 

Before you can get Node and Postgres to play nice, you'll need to know how to use asyncronous functions. This article is a great introduction:

[Learn Callbacks, Promises, and Async/Await in JS by Making Ice Cream](https://www.freecodecamp.org/news/javascript-async-await-tutorial-learn-callbacks-promises-async-await-by-making-icecream/)
```

Notice that the links can be perfectly normal (and user-friendly) text.